src/mnode/ticket-processor.h

```diff
@@ -13,6 +13,8 @@
 #include "mnode/ticket.h"
 
 constexpr int DATASTREAM_VERSION = 1;
+constexpr unsigned char kTicketCompressEnableMask  = 0x80;
+constexpr unsigned char kTicketCompressDisableMask = 0x7f;
```

Comment:
can you please uppercase global constants and also make second one depend on the first one like this:
constexpr unsigned char TICKET_COMPRESS_ENABLE_MASK  = 0x80;
constexpr unsigned char TICKET_COMPRESS_DISABLE_MASK = ~TICKET_COMPRESS_ENABLE_MASK;